thruster/Cargo.toml

```
@@ -18,6 +18,10 @@ debug = true
 name = "most_basic_async"
 required-features = ["thruster_async_await"]
 
+[[example]]
+name = "errors_async"
+required-features = ["thruster_async_await"]
```

This should also require the feature `thruster_error_handling`

```suggestion
required-features = ["thruster_async_await", "thruster_error_handling"]
```